packages/pn-personafisica-webapp/src/component/Contacts/LegalContactsList.tsx

```diff
@@ -101,6 +101,7 @@ const LegalContactsList = ({ recipientId, legalAddresses }: Props) => {
               recipientId={recipientId}
               senderId="default"
               contactType={LegalChannelType.PEC}
+              isVerifyingAddress={(verifyingAddress != null)}
```

Comment:
I should put the condition {verifyingAddress && ... } where DigitalContactElem is used.
Wait for the Norma's reply on the issue, to understand what must be done